lib/perl/Genome/Model/Tools/ImportAnnotation/UpdateAnnotation.pm

```diff
@@ -64,8 +64,8 @@ sub execute
     my $flatfile = $self->derive_flatfile_location();
     my $gbff = "/gscmnt/sata835/info/medseq/annotation_data/human.rna.gbff"; # hard code this for now
 
-    my $ensembl_host = "mysql1"; # or mysql2?
-    my $ensembl_user = "mse";
+    my $ensembl_host = Genome::Config::get('db_ensembl_host');
+    my $ensembl_user = Genome::Config::get('db_ensembl_user');
```

Comment:
Do we need to keep `Genome/Model/Tools/ImportAnnotation` around?  it looks really old.